lib/pace/Makefile

```diff
@@ -1,4 +1,5 @@
 SHELL = /bin/sh
+CMAKE= cmake
```

Comment:
@akohlmey can you wait on this change? I submitted a PR that adds the makefile back in here: https://github.com/ICAMS/lammps-user-pace/pull/19 which doesn't require CMake.

Reply:
Sure. You need the access permissions changed, too, right? So we would need another PR from Yuri before we can finish the release anyway.

Reply:
Right

Reply:
The lib/pace/Makefile changes have been reverted.